transactions/2.0/_index.md

```diff
@@ -5,19 +5,19 @@ summary: "Release for Jakarta EE 9"
 ---
 Jakarta Transactions - Need scope statement
 
-* [Jakarta Transactions 2.0 Release Record]()
+* [Jakarta Transactions 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jta/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
 * [Jakarta Transactions 2.0 Specification Document]() (PDF)
 * [Jakarta Transactions 2.0 Specification Document]() (HTML)
 * [Jakarta Transactions 2.0 Javadoc](./apidocs)
```

Comment:
Somewhere in your apidocs, can you add an identifier for the version that is being rendered?  Some teams have just put the version in the title of the javadoc, Package jakarta.transaction 2.0-RC2.  Others have put the version in the top header on the right hand side of "Package Class Use Tree Deprecated Index Help".  (Sorry, I don't know how to do this latter one.) Some have done both types of version declaration.

Reply:
I have put it at the top of the page (not the title though), I edited a h1.

I have raised https://github.com/eclipse-ee4j/jta-api/issues/123 to try to get this into the build for future.

Thanks!

Reply:
Sounds good.  A couple of other components have had problems with generating the version properly.  No big deal.  Thanks for opening the Issue for tracking and eventual fixing.